cli.js

```diff
@@ -33,6 +34,8 @@ run = () => {
 	menu.recommendCommands();
 	menu.showHelpOnFail();
 	menu.argv;
-}
+
+	globalExceptionHandling;
```

Comment:
Never executed code. The only reason you have listeners for the process is the require of the module you have above